setup.py

```diff
@@ -46,7 +46,7 @@ def collectExtraFiles():
     url="https://github.com/terrapower/armi/",
     license="Apache 2.0",
     long_description=README,
-    python_requres=">=3.6",
+    python_requres=">=3.7",
```

Comment:
Typo in "requires"

Reply:
OMG. I just checked 'git blame' and that typo has been there for at least two years.

Amazing. Yeah, fixing it now.

Reply:
Fixed